packages/typespec-vscode/test/extension/common/common-steps.ts

```diff
@@ -0,0 +1,240 @@
+import { Key, keyboard } from "@nut-tree-fork/nut-js";
+import { rm } from "fs/promises";
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { Locator, Page } from "playwright";
+import { retry, screenshot, sleep } from "./utils";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
you still haven't explained why `import.meta.dirname` is not working?

Reply:
Sorry, it has been updated for now.